packages/@react-native-windows/cli/src/runWindows/utils/msbuildtools.ts

```diff
@@ -95,8 +95,8 @@ export default class MSBuildTools {
 
     // Building projects in parallel increases compiler memory usage and
     // doesn't lead to dramatic performance gains (See #4739). Only enable
-    // parallel builds on machines with >16GB of memory to avoid OOM errors
-    const highMemory = totalmem() > 16 * 1024 * 1024 * 1024;
+    // parallel builds on machines with >15GB of memory to avoid OOM errors
+    const highMemory = totalmem() > 15 * 1024 * 1024 * 1024;
```

Comment:
You could also pass "--singleproc false". At least for older versions, there were some who saw build issues with 16gb machines.  

This maybe should have been testing for free memory instead of total.

Reply:
thanks, I'm still experimenting a a bit here, there is also a /p:MultiTool we could pass to increase paralellism inside the tools for native code.

Reply:
Still struggling for the build to be multicore...

Reply:
Reverted all this in favor of 4c32gb vms